x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/step_rule_actions/translations.tsx

```diff
@@ -5,17 +5,36 @@
  */
 
 import { i18n } from '@kbn/i18n';
+import { startCase } from 'lodash/fp';
 
 export const COMPLETE_WITHOUT_ACTIVATING = i18n.translate(
-  'xpack.siem.detectionEngine.createRule. stepScheduleRule.completeWithoutActivatingTitle',
+  'xpack.siem.detectionEngine.createRule.stepScheduleRule.completeWithoutActivatingTitle',
   {
     defaultMessage: 'Create rule without activating it',
   }
 );
 
 export const COMPLETE_WITH_ACTIVATING = i18n.translate(
-  'xpack.siem.detectionEngine.createRule. stepScheduleRule.completeWithActivatingTitle',
+  'xpack.siem.detectionEngine.createRule.stepScheduleRule.completeWithActivatingTitle',
   {
     defaultMessage: 'Create & activate rule',
   }
 );
+
+export const NO_CONNECTOR_SELECTED = i18n.translate(
+  'xpack.siem.detectionEngine.createRule.stepRuleActions.noConnectorSelectedErrorText',
```

Comment:
I think you'll want these keys as `ErrorMessage` per the  [i18n guidelines](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#defining-type-for-message).
```suggestion
  'xpack.siem.detectionEngine.createRule.stepRuleActions.noConnectorSelectedErrorMessage',
```